com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -11,364 +12,230 @@ namespace MLAPI.Prototyping
     [AddComponentMenu("MLAPI/NetworkTransform")]
     public class NetworkTransform : NetworkBehaviour
     {
-        internal class ClientSendInfo
+        /// <summary>
+        /// Server authority only allows the server to update this transform
+        /// Client authority only allows the client owner to update this transform
+        /// Shared authority allows everyone to update this transform
+        /// </summary>
+        public enum Authority
         {
-            public float LastSent;
-            public Vector3? LastMissedPosition;
-            public Quaternion? LastMissedRotation;
+            Server = 0, // default
+            Client,
+            Shared
         }
 
         /// <summary>
-        /// The base amount of sends per seconds to use when range is disabled
+        /// TODO this will need refactoring
+        /// Specifies who can update this transform
         /// </summary>
-        [Range(0, 120)]
-        public float FixedSendsPerSecond = 20f;
+        [SerializeField, Tooltip("Defines who can update this transform.")]
+        public Authority authority = Authority.Server; // todo Luke mentioned an incoming system to manage this at the NetworkBehaviour level, lets sync on this
```

Comment:
just 2 things (my two cents :P)

1. changes in this file (`NetworkTransform.cs`) makes this PR at least a `feat:` IMHO.
2. can we get all other changes in testproject (such as `PlayerMovement.cs`) without getting changes in this file? if so, let's do that and align on other details internally maybe?

Reply:
Changed to feat
Will merge this soon to develop